CHANGELOG.md

```diff
@@ -6,6 +6,53 @@ in this file.
 Detailed changes to the Solar Forecast Arbiter Core python library can be found
 in the core documentation's [what's new](https://solarforecastarbiter-core.readthedocs.io/en/latest/whatsnew/index.html) series.
 
+
+## [1.0beta5] - Unreleased
+
+### Added
+
+- All report plots are now created with Plotly instead of Bokeh. Plotly's JSON specification and more mature javascript library makes it easier to optimize reports for display in web browsers.
+
+- Added API endpoint to recompute report at `/reports/<report_id>/recompute`. API users can access this endpoint now, and a button will be added to the dashboard in a future release.
+
+- Added API endpoint to get the actions the current user is allowed to take on
+  an object at `/users/actions-on/<object_id>`.
+
+- Added API endpoints to query for latest available data or a timerange of
+  available data. These endpoints are found by appending `/latest` or
+  `/timerange` to the end of a values endpoint.
+
+- Added support for event forecasts and associated reports. A list of supported
+  metrics can be found on the [metrics page](http://localhost:4000/metrics/#metrics-for-deterministic-event-forecasts).
+
+- Metrics now automatically determine normalization based on Observation type.
+  AC power observations are normalized by AC capacity; DC power by DC capacity.
+  Normalized metrics set to nan for all other variables
+
+- Users may now select a reference forecast to use in computing  
+  [forecast skill](http://localhost:4000/metrics/#s).
+
+### Changed
+
+- Individual forecasts or observations can be singled out by double-clicking
+  the legend of a report timeseries or scatter plot.
+
+- Report timeseries no longer draw a line over periods of missing data.
+
+- Improved the speed of reading aggregate values from the API.
+
+- Improved the speed of storing timeseries values at the API.
+
+- Dashboard will not hide buttons for actions that the users cannot take.
```

Comment:
@lboeman I'm working on a PR to your branch. Can you clarify this item. Seems that the first `not` should be removed.

Reply:
You're right, that's a typo.